api/src/lib.rs

```diff
@@ -20,7 +20,7 @@ pub mod metrics;
 mod page;
 mod response;
 pub mod runtime;
-mod set_failpoints;
+pub mod set_failpoints;
```

Comment:
We need to expose this because we don't use `aptos_api::attach_poem_to_runtime`, instead building our own paths with the poem server.

Reply:
Yep